arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,778 @@
+	                  "SPI_4_MISO",      "SPI_4_CS#",       "MAG_N_IRQ",       "FPGA_RESET#",     
+	                  "SPI_3_FPGA_CS#",  "SPI_3_CLK",       "SPI_3_MOSI",      "SPI_3_MISO",      
+	                  "PWM_5",           "",                "",                "ID5",             
+	                  "GYRO_INT",        "6UL_FORCE_5V_ON", "EN_EMMC_3.3V#",   "EN_YEL_LED#",     
+	                  "EN_RED_LED#",     "EN_GRN_LED#",     "EN_BLU_LED",      "FRAM_SPI_CS#",    
+	                  "SD_VSEL_1.8V",    "",                "",                "";                
+};
+
+&gpio5 {
+	gpio-line-names = "POWER_FAIL",     "GPS_PPS_OUT",    "",               "GPIO_DVFS",      
+	                  "",               "SILAB_C2_CLK",   "SILAB_C2_DATA",  "SILAB_C2_RESET", 
+	                  "SPARE_4",        "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "",               
+	                  "",               "",               "",               "";               
+};
+
+&i2c1 {
+	clock-frequency = <100000>;
+	pinctrl-names = "default", "gpio";
+	pinctrl-0 = <&pinctrl_i2c1>;
+	pinctrl-1 = <&pinctrl_i2c1_gpio>;
+	scl-gpios = <&gpio1 2 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	sda-gpios = <&gpio1 3 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+	status = "okay";
+
+	/*
+         * Reference: Documentation/devicetree/bindings/iio/st-sensors.txt
+	 *
+	 *   - MAG_N_IRQ enters NAND_ALE/SD2_RST#/PWM3 pad B4
+	 *   - as well as the FPGA
+	 */
+	st_magn: magnetometer@1e {
+		compatible = "st,lis2mdl";
+		reg = <0x1e>;
+		st,drdy-int-pin = <1>;
+		interrupt-parent = <&gpio4>;
+		interrupts = <10 IRQ_TYPE_LEVEL_HIGH>;    /* See MX6UL_PAD_NAND_ALE__GPIO4_IO10 */
+	};
+
+	supervisor: wdt@54 {
+		compatible = "technologic,ts7100-wdt";
+		enable-early;
+		/* 5min timeout default, in case of slow userspace.
+		 * Set to 0 to disable WDT at startup. Userspace procs
+		 * can still start feeding with different values later -
+                 * watchdog(8) defaults to 60 seconds.
+		 */
+		timeout-sec = <300>;
+		reg = <0x54>;
+
+		/*
+		 * Not using uC WDT by default any longer. If you need features
+		 * provided by this, such as a real poweroff state, an external
+		 * WDT, or proper feedback from the uC on the platform's reboot
+		 * source (via tsmicroctl), then enable this driver (and disable
+		 * the CPU driver, wdog1,  unless you want both active).
+		 */
+
+		status = "disabled";
+	};
+
+	m41t00s: rtc@68 {
+		compatible = "st,m41t00";
+		reg = <0x68>;
+	};
+
+	/*
+	 * Reference: Documentation/devicetree/bindings/iio/imu/st_lsm6dsx.txt
+	 *
+	 *   - GYRO_INT enters CSI_HSYNC (pad F3)
+	 */
```

Comment:
Remove this, not necessary. The reference can be found by looking at the compatible name. The pin itself can be looked up via the pinctrl table.